components/StandardListItem/StandardListItemStyles.scss

```diff
@@ -0,0 +1,52 @@
+.StandardListItem {
+
+  display: flex;
+  align-items: center;
+
+  text-align: center;
+
+  width: 100%;
+
+  padding: 20px 15px;
```

Comment:
Can you please change padding to 15px from top/bottom and 20px from left/right? Further, for mobile (<768px) view, please change it to 12px from top/bottom and 15px from left/right.